packages/styled-components/src/constructors/createGlobalStyle.js

```diff
@@ -14,6 +14,8 @@ import type { Interpolation } from '../types';
 
 type GlobalStyleComponentPropsType = Object;
 
+const seen = new Set();
```

Comment:
I think this will warn excessively for SSR, which is undesirable. It's also possible and OK to mount 2 instances of the same global style which would trigger this warning.